MapboxNavigation/MapboxSpeechSynthesizer.swift

```diff
@@ -37,8 +37,14 @@ open class MapboxSpeechSynthesizer: NSObject, SpeechSynthesizing {
      */
     public var audioPlayer: AVAudioPlayer?
     
+    /**
+     Mapbox speech engine instance.
+     
+     `MapboxSpeechSynthesizer` uses it to convert insturctions text to audio.
+     */
+    public private(set) var speechEngine: SpeechSynthesizer
```

Comment:
Great idea to name this property something more descriptive than `speech`. The distinction between a “speech synthesizer” and a “speech engine” may be fuzzy for developers, so consider calling it `remoteSpeechSynthesizer` or `networkSpeechSynthesizer`.